wasp/draw565.py

```diff
@@ -518,3 +518,43 @@ def darken(self, color, step=1):
         b = bm - step if bm > step else 0
 
         return (r | g | b)
+
+    def bezier(self, color, precision, control_points):
+        """Draw a bezier curve using control points and specific precision.
+
+        The precision determines how many samples along the line will be taken.
+        When precision is higher, the function is *much* slower, but also draws nicer lines.
+        Control points define the curve, and should be a list of tuples of (x, y) coordinates.
+
+        Example:
+
+        .. code-block:: python
+
+            draw.bezier(0xf800, 100, [(0, 0), (1, 1), (2, 0)])
+
+        :param color:           A RGB565 color in which the curve will be drawn.
```

Comment:
Put color at the end of parameters like other func ?

Reply:
Color is at the start here somewhat intentionally - the list of points can get quite long, imo it makes a bit more sense to break that convention here.